tests/foreman/cli/test_fact.py

```diff
@@ -27,7 +28,8 @@
         'os::family',
         'system_uptime::seconds',
         'memory::system::total',
-        'networking::ip6' if settings.server.is_ipv6 else 'networking::ip',
+        # TODO(gtalreja): How should we handle dualstack here?
+        'networking::ip6' if settings.server.network_type == NetworkType.IPV6 else 'networking::ip',
```

Comment:
I think this should be work as this is updated now, as `networking::ip` fact should exist for dualstack too